admin/static/js/metrics/metrics.js

```diff
@@ -1,187 +1,499 @@
-"use_strict";
+'use strict';
 
+require('c3/c3.css');
+require('keen-dataviz/dist/keen-dataviz.min.css');
 
+
+var c3 = require('c3/c3.js');
 var keen = require('keen-js');
+var keenDataviz = require('keen-dataviz');
+var keenAnalysis = require('keen-analysis');
 
 
-var client = new Keen({
+var client = new keenAnalysis({
     projectId: keenProjectId,
-    readKey : keenReadKey
+    readKey: keenReadKey
 });
 
+var publicClient = new keenAnalysis({
+    projectId: keenPublicProjectId,
+    readKey: keenPublicReadKey
+});
 
-Keen.ready(function () {
+var defaultHeight = 200;
```

Comment:
Works in my versions of firefox and chrome, had to update to the latest version of Safari for some of the fancier javascript to work properly...